SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -429,18 +429,21 @@ FRepChangeState USpatialActorChannel::CreateInitialRepChangeState(TWeakObjectPtr
 	for (uint16 CmdIdx = 0; CmdIdx < CmdCount; ++CmdIdx)
 	{
 		const FRepLayoutCmd& Cmd = Replicator.RepLayout->Cmds[CmdIdx];
-		const FRepChangedParent& Parent = Replicator.RepState->GetSendingRepState()->RepChangedPropertyTracker->Parents[Cmd.ParentIndex];
```

Comment:
Can't we do a safe get instead and return on null instead? Still seems a bit brittle currently. We could even add a couple of ensure to make sure some combo's that have to be valid are valid.